docs/installation/docker/site-template/setup.md

```diff
@@ -31,14 +31,40 @@ This documentation assumes you will be building your production image on the pro
 
 ## Adding a Staging Site
 
-The process for setting up a staging site is the same as production, but you will need to change the `DOMAIN` variable in your `.env` file on your staging server, to contain the URL for your staging site.
+The process for setting up a staging site is the same as production, but you will need to use a different URL. Since the URL is set in the .env file, which is checked into your git repository, you may wish to use docker-compose.override.yml for this. In docker-compose.override.yml you will need to override anywhere the `DOMAIN` variable is used, for example:
```

Comment:
Just wrap some of the strings in backticks. Better readability, perhaps? Dunno.

```suggestion
The process for setting up a staging site is the same as production, but you will need to use a different URL. Since the URL is set in the `.env` file, which is checked into your git repository, you may wish to use `docker-compose.override.yml` for this. In `docker-compose.override.yml` you will need to override anywhere the `DOMAIN` variable is used, for example:
```